recipes-devtools/deepstream/deepstream-tlt-apps_3.0.bb

```diff
@@ -0,0 +1,37 @@
+DESCRIPTION = "NVIDIA Transfer Learning Toolkit sample applications"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=dbef1fb16cd9e5c5249a8e0c5e639fb0"
+
+DEPENDS = " deepstream-5.0"
+
+SRC_URI = "git://github.com/NVIDIA-AI-IOT/deepstream_tao_apps.git;branch=release/tlt3.0"
+SRCREV = "16405b1a725a888cf6473cdd4fd0cb7effbf393f"
+
+S = "${WORKDIR}/git"
+
+inherit pkgconfig cuda
```

Comment:
The recipe needs a
```
COMPATIBLE_MACHINE = "(tegra)"
```
line to prevent it from being parsed for non-tegra builds.